projects/logging/application.ts

```diff
@@ -42,6 +42,9 @@ export const createApp = (): express.Application => {
                 processLog(data)
                 res.send('Log success')
             } else {
+                logger.error(
```

Comment:
Is this going to cause a spike in sentry again?

Reply:
No as this is a backend only change - it will result in more logs going to cloudwatch - one for every time the logging service receives a request which is too large. Possibly not something necessary forever but it would be quite useful at the moment.